packages/schema/index.ts

```diff
@@ -0,0 +1,47 @@
+export default {
+  'link': {
+    type: 'inline',
+    attributes: [
+      'destination',
+      'title'
+    ]
+  },
+  'paragraph': {
+    type: 'paragraph'
+  },
+  'heading': {
+    type: 'block',
+    attributes: [
+      'level'
+    ]
+  },
+  'item': {
+    type: 'block'
+  },
+  'list': {
+    type: 'block',
+    attributes: [
+      'type',
+      'startsAt'
```

Comment:
this should probably also have "level" or "nestLevel"

Reply:
Should this be implicit through nested annotations? I'm not sure it makes sense otherwise.